curriculum/challenges/english/25-front-end-development/quiz-javascript-loops/66edcd49e73385dd4df54ac7.md

```diff
@@ -17,439 +17,477 @@ Answer all of the questions below correctly to pass the quiz.
 
-Placeholder distractor 1
+The `continue` statement
 
 ---
 
-Placeholder distractor 2
+The `label` statement
 
 ---
 
-Placeholder distractor 3
+The `skip` statement
 
 #### --answer--
 
-Placeholder answer
+The `break` statement
 
 ### --question--
 
 #### --text--
 
-Placeholder question
+Which statement can be used to skip a current iteration and move to the next iteration?
 
 #### --distractors--
 
-Placeholder distractor 1
+The `break` statement
 
 ---
 
-Placeholder distractor 2
+The `label` statement
 
 ---
 
-Placeholder distractor 3
+The `return` statement
 
 #### --answer--
 
-Placeholder answer
+The `continue` statement
 
 ### --question--
 
 #### --text--
 
-Placeholder question
+It is generally recommended to avoid using the `for...in` loop in JavaScript. What is the reason for that?
 
 #### --distractors--
 
-Placeholder distractor 1
+It only works with arrays and not with objects.
 
 ---
 
-Placeholder distractor 2
+It is the best way to iterate over large objects.
 
 ---
 
-Placeholder distractor 3
+It guarantees the order of property enumeration.
 
 #### --answer--
 
-Placeholder answer
+It could enumerate over inherited properties of an object.
 
 ### --question--
 
 #### --text--
 
-Placeholder question
+What will be the output of the code below?
+
+```javascript
+for (let i = 1; i < 6; i++) {
+  if (i === 4) break;
+  console.log(i);
+}
+```
 
 #### --distractors--
 
-Placeholder distractor 1
+1,2,3,4,5,6
 
 ---
 
-Placeholder distractor 2
+1,2,3,4,5
 
 ---
 
-Placeholder distractor 3
+1,2,3,4
 
 #### --answer--
 
-Placeholder answer
+1,2,3
 
 ### --question--
 
 #### --text--
 
-Placeholder question
+What will be the output of the code below?
+
+```javascript
+const shoppingList = { tomatoes: 4, apples: 10 };
+for (const item in shoppingList) {
+  console.log(item);
+}
+```
 
 #### --distractors--
 
-Placeholder distractor 1
+tomatoes - 4, apples - 10
 
 ---
 
-Placeholder distractor 2
+4, 10
 
 ---
 
-Placeholder distractor 3
+tomatoes:4, apples:10
 
 #### --answer--
 
-Placeholder answer
+tomatoes, apples
 
 ### --question--
 
 #### --text--
 
-Placeholder question
+What will be the output of the code below?
+
+```javascript
+for (let i = 2; i <= 13; i++) {
+  if (i % 2 === 0) continue;
+  console.log(i);
+}
+```
 
 #### --distractors--
 
-Placeholder distractor 1
+2,4,6,8,10,12
 
 ---
 
-Placeholder distractor 2
+1,3,5,7,9,11,13
 
 ---
 
-Placeholder distractor 3
+1,3,5,7,9,11
 
 #### --answer--
 
-Placeholder answer
+3,5,7,9,11,13
 
 ### --question--
 
 #### --text--
 
-Placeholder question
+What would be the output of the code below?
+
+```javascript
+const fruits = ["Mango", "Pineapple", "Oranges"];
+for (const fruit in fruits) {
+  console.log(fruit);
+}
+```
 
 #### --distractors--
 
-Placeholder distractor 1
+Mango, Pineapple
 
 ---
 
-Placeholder distractor 2
+Pineapple, Oranges, Mango
 
 ---
 
-Placeholder distractor 3
+Oranges, Pineapple, Mango
 
 #### --answer--
 
-Placeholder answer
+Mango, Pineapple, Oranges
```

Comment:
These refer to specific terminal outputs, and need to be in code tags.

Reply:
should I have them in individual lines too?

Reply:
For clarity, is this what you mean
![image](https://github.com/user-attachments/assets/c0d38efe-c8ac-48cd-a972-310df7e8a925)